core/tauri-runtime/src/webview.rs

```diff
@@ -97,6 +99,13 @@ impl WebviewAttributes {
     self.additional_browser_args = Some(additional_args.to_string());
     self
   }
+
+  /// Enables Incognito/InPrivate mode for the WebView.
+  #[must_use]
+  pub fn enable_incognito(mut self) -> Self {
```

Comment:
```suggestion
  pub fn incognito(mut self) -> Self {
```

Reply:
Not exactly sure where it should be exposed in config but if I did it right it should be exposed in JS

Reply:
For it to be exposed in config, you need to add it here https://github.com/tauri-apps/tauri/blob/942249060ed12a5d21a2b21c30e0638c1d2b9df0/core/tauri-utils/src/config.rs#L695 and for JS you need to add it to https://github.com/tauri-apps/tauri/blob/942249060ed12a5d21a2b21c30e0638c1d2b9df0/tooling/api/src/window.ts#L2097 
and for both to work you need to modify https://github.com/tauri-apps/tauri/blob/942249060ed12a5d21a2b21c30e0638c1d2b9df0/core/tauri-runtime-wry/src/lib.rs#L699

Reply:
Yeah I figured that out (if im right)